docs/gettingstarted/quickstart.ipynb

```diff
@@ -2,24 +2,175 @@
  "cells": [
```

Comment:
<p>I would change <code>In this case</code> to something more like <code></code>Becuase we are downloading from the web source we need to install an additional package....<code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>